### ISSUE TITLE
[A11y] Fix color contrast of Avalon release text in the footer

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/avalon#6352]
### Description

Color contrast of the Avalon release text in the footer is 2.84:1. This value does not meet the minimum requirement stated by WCAG 2.1, 1.4.3 success criterion. This is a Level AA violation.

### Done Looks Like
- [ ] Fix color contrast of the Avalon release text

**Comment by joncameron** [comment]
@Dananji it looks like this change is being overridden by https://github.com/avalonmediasystem/avalon/blob/ba9668892ec7da05cad58aeb42074ed92a10d389/app/assets/stylesheets/avalon/_homepage.scss#L113-L128; the text is still #FFFFFF on avalon-dev.

**Comment by Dananji** [comment]
@joncameron This color contrast adjustment for the footer's release text has been implemented on all pages except the homepage. For the homepage, where the footer's background is green, the current CSS in the highlighted code appears to be providing the expected color contrast ratio. 
I tested the homepage using the ANDI bookmarklet and it did not indicate any color contrast concerns related to this specific CSS.
